lib/web_ui/lib/src/ui/window.dart

```diff
@@ -257,3 +257,23 @@ class FrameData {
 
   final int frameNumber;
 }
+
+class GestureSettings {
+  const GestureSettings({this.physicalTouchSlop});
+
+  final double? physicalTouchSlop;
+
+  @override
+  bool operator ==(Object other) {
+    if (other.runtimeType != runtimeType)
+      return false;
```

Comment:
```suggestion
    if (other.runtimeType != runtimeType) {
      return false;
    }
```

for consistency with the rest of this file

Reply:
done